File: baekjoon/CLASS_1++/020-10818-최소최대.py
import sys
input = sys.stdin.readline

N = int(input())
numbers = list(map(int, input().split(' ')))
maxNum = numbers[0]
minNum = numbers[0]
for num in numbers :
    if num > maxNum :
        maxNum = num
    elif num < minNum : 
        minNum = num

print(minNum, maxNum)

File: baekjoon/CLASS_1++/020-10818-최소최대.js
let [N, arr] = require('fs').readFileSync("test.txt").toString().trim().split('\n');
N = +N;
const numbers = arr.split(' ').map(Number);
let minNum = numbers[0];
let maxNum = numbers[0];
numbers.forEach((num) => {
    if (num > maxNum) maxNum = num;
    else if (num < minNum) minNum = num;
})

console.log(minNum, maxNum);